File: web3rpc/rpc-specs/code-samples/python/eth/account/getCode.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x1cbd3b2770909d4e10f157cabc84c7264073c9ec"
blockNumberOrHash = "latest"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_code(address, blockNumberOrHash)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/account/getCode.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const address = '0xce9fba8dca42d096d019cc1cb89f5803a2c40fb3'
    const blockNumberOrHash = '0x2'
    
    sdk.eth.getCode(address, blockNumberOrHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()